docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1269 @@
+
+    <para>The subsequent sections will examine some of these more closely and
+    why each of them is there.</para>
+
+    <sect2 id="ValYAML_STorage">
+      <title>Storage</title>
+
+      <para>Containerized Storage is another key concept that differs from
+      bare metal. There are a few differences between container and bare metal
+      storage. The Storage section is fairly well defined between the schema
+      file, and the <emphasis>values.yaml</emphasis>. A good approach towards
+      storage is to clearly understand your storage needs, and to outline
+      them, and once you have that basic structure in mind the schema can help
+      to fill in the details. The schema should have a decent description for
+      each attribute. All storage should be defined via planes. There is a
+      relevant comment in the <emphasis>values.yaml</emphasis> further
+      describing storage.</para>
+
+      <programlisting>## storage:
+##
+## 1. If an engine component has the dataPlane property set, 
+#       then that plane will be the default data location for that component.
+## 2. If there is a plane definition with a category of "data" 
+#       then the first matching plane will be the default data location
+##
+## If a data plane contains the storageClass property then an implicit pvc 
+#       will be created for that data plane.
+##
+## If plane.pvc is defined, a Persistent Volume Claim must exist with that name, 
+#       storageClass and storageSize are not used.
+##
+## If plane.storageClass is defined, storageClassName: &lt;storageClass&gt;
+## If set to "-", storageClassName: "", which disables dynamic provisioning
+## If set to "", choosing the default provisioner.  
+#       (gp2 on AWS, standard on GKE, AWS &amp; OpenStack)
+##
+## plane.forcePermissions=true is required by some types of provisioned
+## storage, where the mounted filing system has insufficient permissions to be
+## read by the hpcc pods. Examples include using hostpath storage (e.g. on
+## minikube and docker for desktop), or using NFS mounted storage.
+</programlisting>
+
+      <para>There are different categories of storage, for an HPCC Systems
+      deployment you must have at a minimum a dali category, a dll category,
+      and at least 1 data category. These types are generally applicable for
+      every configuration in addition to other optional categories of
+      data.</para>
+
+      <para>All storage should be in a storage plane definition. This is best
+      described in the comment in the storage definition in the values
+      file.</para>
+
+      <programlisting>planes:
+  #   name: &lt;required&gt;
+  #   prefix: &lt;path&gt;                        # Root directory for accessing the plane 
+                                            # (if pvc defined), 
+  #                                         # or url to access plane.
+  #   category: data|dali|lz|dll|spill|temp # What category of data is stored on this plane?
```

Comment:
this should updated for 8.6, where "git" and "remote" will now be plane types too.